### PR TITLE
Fix lilac.yaml

### PR DESCRIPTION
**Pull request by SolarAquarion**

### CODE REVIEW

**Comment by SolarAquarion** [comment]
@yuyichao i hope you can squash and fix this.  Because i have something but like whatever

File: archlinuxcn/auracle-git/lilac.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 maintainers:
   - github: colinkeenan
+  - github: solaraquarion 
 
-managed: false
+ build_prefix: extra-x86_64
```
Comment (yuyichao) [inline archlinuxcn/auracle-git/lilac.yaml:5]: Is yaml sensitive to leading space like this?

File: archlinuxcn/auracle-git/lilac.yaml [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+ 
+ 
+update_on: vcs
```
Comment (yuyichao) [inline archlinuxcn/auracle-git/lilac.yaml:8]: And I think this is old syntax?

**Review by yuyichao — COMMENTED** [review]
Commit structure looks good currently, not sure if there's anything random before the last force push. The commit and PR title should be slightly more descriptive. Not that important here but you should at least mention turn on auto build or sth like that...

Since this isn't my package I'll at least wait a few days before merging...

**Comment by SolarAquarion** [comment]
It is

**Comment by SolarAquarion** [comment]
@yuyichao yo

**Comment by imlonghao** [comment]
Invalid YAML format.

https://github.com/archlinuxcn/repo/pull/2116/checks?check_run_id=1989237698

**Comment by SolarAquarion** [comment]
fixed it i think

**Comment by yuyichao** [comment]
> @yuyichao yo

Yes, and I was waiting for the fix.
Seems that the file has end-of-line whitespace and the indentation is a little confusing but it seems that it is at least valid now.

**Pull request merged**